Assets/Mirror/Tests/Editor/Weaver/WeaverNetworkBehaviourTests~/NetworkBehaviourGenericSyncVar.cs

```diff
@@ -0,0 +1,12 @@
+using Mirror;
+
+namespace WeaverNetworkBehaviourTests.NetworkBehaviourGeneric
+{
+    class NetworkBehaviourGeneric<T> : NetworkBehaviour
```

Comment:
this test is supposed to be weaved successfully?
if yes, need to move this outside the ~folder name into WeaverNetworkBehaviourTests_IsSuccess